composer.json

```diff
@@ -17,7 +17,7 @@
     "issues": "https://github.com/bueltge/wordpress-multisite-enhancements/issues"
   },
   "require": {
-    "php": ">=5.3",
+    "php": ">=7.2",
```

Comment:
The plugin must run in php >=5.6, so that we need this value. New version are recomandation, but not a must. So pleae change back to min. 5.6. The support required min. 5.6, so this value is ok.